api/Models/Game.cs

```diff
@@ -5,8 +5,11 @@ public class Game
         public string Id { get; set; } = string.Empty;
         
         public string Title { get; set; } = string.Empty;
+        public string? JapaneseTitle { get; set; } = string.Empty;
+        public string? RomanizedTitle { get; set; } = string.Empty;
 
         public DateTime? OriginalReleaseDate { get; set; }
+        public DateTime? MostRecentReleaseDate { get; set; }
```

Comment:
Maybe we could move this into a child object for the release, where it's like
```
Release
{
   Platform
   ReleaseDate
   Regions
}
```

Reply:
Might be good idea, need to discuss further